ga4gh/cli.py

```diff
@@ -547,18 +632,44 @@ def addVariantSearchOptions(parser):
     addPageSizeArgument(parser)
 
 
+def addAnnotationsSearchOptions(parser):
+    """
+    Adds common options to a annotation searches command line parser.
+    """
+    addAnnotationSetIdArgument(parser)
+    addReferenceNameArgument(parser)
+    addReferenceIdArgument(parser)
+    addStartArgument(parser)
+    addEndArgument(parser)
+    addFeatureIdsArgument(parser)
+    addEffectsArgument(parser)
+    addPageSizeArgument(parser)
+
+
 def addVariantSetIdArgument(parser):
     parser.add_argument(
         "--variantSetId", "-V", default=None,
         help="The variant set id to search over")
 
 
+def addAnnotationSetIdArgument(parser):
+    parser.add_argument(
+        "--variantAnnotationSetId", "-V", default=None,
+        help="The annotation set id to search over")
+
+
 def addReferenceNameArgument(parser):
     parser.add_argument(
         "--referenceName", "-r", default="1",
         help="Only return variants on this reference.")
 
 
+def addReferenceIdArgument(parser):
+    parser.add_argument(
+        "--referenceId", "-rid", default="",
```

Comment:
Short form arguments must  be a single letter for POSIX --- just pick an unused one.